[2025-03-01T23:39:23.944+0000] {processor.py:186} INFO - Started process (PID=17947) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:39:23.947+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:39:23.949+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:23.949+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:39:23.989+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:23.989+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:39:23.990+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:23.990+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:39:23.992+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:23.992+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:39:23.993+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:23.993+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:39:24.438+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.438+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:39:24.438+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.438+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:39:24.439+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.439+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:39:24.439+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.439+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:39:24.803+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:39:24.906+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.905+0000] {override.py:1900} INFO - Created Permission View: can read on DAG:Lab1_ETL
[2025-03-01T23:39:24.913+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.913+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG:Lab1_ETL
[2025-03-01T23:39:24.915+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.915+0000] {override.py:1900} INFO - Created Permission View: can edit on DAG:Lab1_ETL
[2025-03-01T23:39:24.916+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.916+0000] {override.py:1900} INFO - Created Permission View: can read on DAG Run:Lab1_ETL
[2025-03-01T23:39:24.918+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.918+0000] {override.py:1900} INFO - Created Permission View: menu access on DAG Run:Lab1_ETL
[2025-03-01T23:39:24.920+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.920+0000] {override.py:1900} INFO - Created Permission View: can create on DAG Run:Lab1_ETL
[2025-03-01T23:39:24.923+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.923+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG Run:Lab1_ETL
[2025-03-01T23:39:24.925+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.924+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:39:24.934+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.934+0000] {dag.py:3252} INFO - Creating ORM DAG for Lab1_ETL
[2025-03-01T23:39:24.946+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:24.945+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-02-28 02:30:00+00:00, run_after=2025-03-01 02:30:00+00:00
[2025-03-01T23:39:24.954+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 1.028 seconds
[2025-03-01T23:39:32.118+0000] {processor.py:186} INFO - Started process (PID=17957) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:39:32.123+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:39:32.126+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:32.125+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:39:32.167+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:32.167+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:39:32.173+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:32.172+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:39:32.174+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:32.174+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:39:32.174+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:32.174+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:39:32.486+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:32.486+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:39:32.487+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:32.487+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:39:32.487+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:32.487+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:39:32.487+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:32.487+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:39:32.843+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:39:32.850+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:32.849+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:39:32.876+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:32.876+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-02-28 02:30:00+00:00, run_after=2025-03-01 02:30:00+00:00
[2025-03-01T23:39:32.889+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.781 seconds
[2025-03-01T23:39:41.883+0000] {processor.py:186} INFO - Started process (PID=17988) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:39:41.884+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:39:41.885+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:41.885+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:39:41.910+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:41.910+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:39:41.912+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:41.912+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:39:41.913+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:41.913+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:39:41.914+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:41.913+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:39:42.217+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:42.216+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:39:42.217+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:42.217+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:39:42.218+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:42.218+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:39:42.218+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:42.218+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:39:42.522+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:39:42.527+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:42.527+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:39:42.559+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:39:42.559+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-02-28 02:30:00+00:00, run_after=2025-03-01 02:30:00+00:00
[2025-03-01T23:39:42.569+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.693 seconds
[2025-03-01T23:40:13.260+0000] {processor.py:186} INFO - Started process (PID=18024) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:40:13.263+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:40:13.264+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:13.264+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:40:13.292+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:13.292+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:40:13.294+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:13.293+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:40:13.295+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:13.295+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:40:13.295+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:13.295+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:40:13.676+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:13.676+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:40:13.677+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:13.677+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:40:13.677+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:13.677+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:40:13.677+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:13.677+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:40:14.065+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:40:14.154+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:14.154+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:40:14.166+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:14.166+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-02-28 02:30:00+00:00, run_after=2025-03-01 02:30:00+00:00
[2025-03-01T23:40:14.175+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.927 seconds
[2025-03-01T23:40:44.527+0000] {processor.py:186} INFO - Started process (PID=18110) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:40:44.529+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:40:44.530+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:44.529+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:40:44.554+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:44.554+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:40:44.555+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:44.555+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:40:44.556+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:44.556+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:40:44.556+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:44.556+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:40:45.033+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:45.032+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:40:45.033+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:45.033+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:40:45.033+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:45.033+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:40:45.034+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:45.034+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:40:45.334+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:40:45.416+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:45.416+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:40:45.426+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:45.425+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:40:45.432+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.917 seconds
[2025-03-01T23:40:58.695+0000] {processor.py:186} INFO - Started process (PID=18120) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:40:58.696+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:40:58.697+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:58.697+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:40:58.710+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:58.710+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:40:58.710+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:58.710+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:40:58.711+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:58.711+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:40:58.711+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:58.711+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:40:59.026+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:59.026+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:40:59.027+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:59.027+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:40:59.027+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:59.027+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:40:59.027+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:59.027+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:40:59.395+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:40:59.401+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:59.401+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:40:59.426+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:40:59.426+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:40:59.439+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.747 seconds
[2025-03-01T23:41:29.491+0000] {processor.py:186} INFO - Started process (PID=18154) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:41:29.492+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:41:29.493+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:41:29.492+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:41:29.514+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:41:29.514+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:41:29.515+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:41:29.515+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:41:29.516+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:41:29.515+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:41:29.516+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:41:29.516+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:41:29.849+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:41:29.848+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:41:29.849+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:41:29.849+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:41:29.850+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:41:29.850+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:41:29.850+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:41:29.850+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:41:30.173+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:41:30.215+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:41:30.215+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:41:30.223+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:41:30.223+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:41:30.231+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.746 seconds
[2025-03-01T23:42:00.611+0000] {processor.py:186} INFO - Started process (PID=18190) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:42:00.615+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:42:00.616+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:00.616+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:42:00.632+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:00.631+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:42:00.632+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:00.632+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:42:00.633+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:00.633+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:42:00.633+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:00.633+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:42:01.005+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:01.005+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:42:01.006+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:01.006+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:42:01.006+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:01.006+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:42:01.006+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:01.006+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:42:01.262+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:42:01.317+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:01.317+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:42:01.326+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:01.326+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:42:01.332+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.733 seconds
[2025-03-01T23:42:31.542+0000] {processor.py:186} INFO - Started process (PID=18223) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:42:31.549+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:42:31.550+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:31.549+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:42:31.590+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:31.590+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:42:31.592+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:31.592+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:42:31.594+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:31.594+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:42:31.595+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:31.594+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:42:32.056+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:32.055+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:42:32.056+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:32.056+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:42:32.057+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:32.057+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:42:32.057+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:32.057+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:42:32.381+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:42:32.486+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:32.486+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:42:32.504+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:42:32.504+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:42:32.516+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.987 seconds
[2025-03-01T23:43:03.099+0000] {processor.py:186} INFO - Started process (PID=18259) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:43:03.102+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:43:03.103+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:03.103+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:43:03.133+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:03.133+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:43:03.134+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:03.134+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:43:03.136+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:03.136+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:43:03.136+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:03.136+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:43:03.597+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:03.597+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:43:03.598+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:03.597+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:43:03.598+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:03.598+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:43:03.598+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:03.598+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:43:03.904+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:43:03.992+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:03.992+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:43:04.008+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:04.008+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:43:04.022+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.933 seconds
[2025-03-01T23:43:34.156+0000] {processor.py:186} INFO - Started process (PID=18301) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:43:34.157+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:43:34.158+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:34.158+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:43:34.184+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:34.184+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:43:34.188+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:34.187+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:43:34.189+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:34.189+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:43:34.189+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:34.189+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:43:34.699+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:34.699+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:43:34.700+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:34.700+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:43:34.700+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:34.700+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:43:34.701+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:34.700+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:43:34.993+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:43:35.066+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:35.065+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:43:35.083+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:43:35.083+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:43:35.093+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.947 seconds
[2025-03-01T23:44:05.472+0000] {processor.py:186} INFO - Started process (PID=18332) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:44:05.475+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:44:05.476+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:05.475+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:44:05.498+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:05.497+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:44:05.498+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:05.498+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:44:05.499+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:05.499+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:44:05.499+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:05.499+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:44:05.948+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:05.948+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:44:05.949+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:05.948+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:44:05.950+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:05.950+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:44:05.951+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:05.950+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:44:06.255+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:44:06.327+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:06.327+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:44:06.337+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:06.337+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:44:06.344+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.880 seconds
[2025-03-01T23:44:36.961+0000] {processor.py:186} INFO - Started process (PID=18363) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:44:36.962+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:44:36.963+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:36.963+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:44:36.987+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:36.987+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:44:36.988+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:36.988+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:44:36.989+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:36.989+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:44:36.990+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:36.990+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:44:37.450+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:37.450+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:44:37.451+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:37.451+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:44:37.452+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:37.452+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:44:37.452+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:37.452+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:44:37.870+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:44:37.941+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:37.941+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:44:37.952+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:44:37.952+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:44:37.959+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 1.010 seconds
[2025-03-01T23:45:08.225+0000] {processor.py:186} INFO - Started process (PID=18401) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:45:08.226+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:45:08.228+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:08.227+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:45:08.250+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:08.250+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:45:08.252+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:08.252+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:45:08.253+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:08.253+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:45:08.253+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:08.253+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:45:08.719+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:08.719+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:45:08.720+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:08.720+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:45:08.720+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:08.720+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:45:08.720+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:08.720+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:45:09.063+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:45:09.134+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:09.134+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:45:09.144+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:09.144+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:45:09.151+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.934 seconds
[2025-03-01T23:45:39.315+0000] {processor.py:186} INFO - Started process (PID=18432) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:45:39.317+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:45:39.318+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:39.318+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:45:39.344+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:39.344+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:45:39.345+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:39.345+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:45:39.346+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:39.346+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:45:39.346+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:39.346+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:45:39.981+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:39.981+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:45:39.982+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:39.982+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:45:39.982+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:39.982+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:45:39.982+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:39.982+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:45:40.470+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:45:40.538+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:40.538+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:45:40.548+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:45:40.548+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:45:40.556+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 1.250 seconds
[2025-03-01T23:46:10.998+0000] {processor.py:186} INFO - Started process (PID=18463) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:11.001+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:46:11.003+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:11.002+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:11.044+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:11.043+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:46:11.045+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:11.045+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:46:11.047+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:11.047+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:46:11.047+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:11.047+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:46:11.509+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:11.509+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:46:11.510+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:11.510+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:46:11.510+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:11.510+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:46:11.510+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:11.510+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:46:11.806+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:11.895+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:11.895+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:46:11.914+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:11.914+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:46:11.921+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.934 seconds
[2025-03-01T23:46:15.637+0000] {processor.py:186} INFO - Started process (PID=18468) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:15.638+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:46:15.639+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:15.639+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:15.659+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:15.659+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:46:15.659+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:15.659+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:46:15.660+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:15.660+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:46:15.660+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:15.660+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:46:15.990+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:15.994+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:15.994+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:46:16.011+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:16.010+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:46:16.020+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.390 seconds
[2025-03-01T23:46:17.012+0000] {processor.py:186} INFO - Started process (PID=18473) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:17.015+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:46:17.016+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:17.016+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:17.040+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:17.040+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:46:17.041+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:17.041+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:46:17.043+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:17.043+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:46:17.043+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:17.043+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:46:17.377+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:17.382+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:17.382+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:46:17.399+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:17.399+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:46:17.409+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.404 seconds
[2025-03-01T23:46:47.841+0000] {processor.py:186} INFO - Started process (PID=18520) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:47.843+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:46:47.845+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:47.844+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:47.872+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:47.871+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:46:47.873+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:47.873+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:46:47.874+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:47.874+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:46:47.875+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:47.874+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:46:48.480+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:46:48.542+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:48.542+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:46:48.551+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:46:48.551+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:46:48.560+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.726 seconds
[2025-03-01T23:47:16.702+0000] {processor.py:186} INFO - Started process (PID=18551) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:47:16.704+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:47:16.706+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:16.705+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:47:16.737+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:16.736+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:47:16.738+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:16.738+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:47:16.739+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:16.739+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:47:16.739+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:16.739+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:47:17.213+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:47:17.217+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:17.217+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:47:17.240+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:17.240+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:47:17.252+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.560 seconds
[2025-03-01T23:47:42.687+0000] {processor.py:186} INFO - Started process (PID=18582) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:47:42.688+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:47:42.690+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:42.689+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:47:42.721+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:42.721+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:47:42.723+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:42.723+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:47:42.724+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:42.724+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:47:42.724+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:42.724+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:47:43.148+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:47:43.210+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:43.210+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:47:43.221+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:47:43.221+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:47:43.230+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.553 seconds
[2025-03-01T23:48:13.586+0000] {processor.py:186} INFO - Started process (PID=18613) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:48:13.588+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:48:13.589+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:13.588+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:48:13.616+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:13.616+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:48:13.617+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:13.617+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:48:13.618+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:13.618+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:48:13.618+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:13.618+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:48:14.063+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:48:14.119+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:14.119+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:48:14.128+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:14.128+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:48:14.134+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.558 seconds
[2025-03-01T23:48:28.247+0000] {processor.py:186} INFO - Started process (PID=18638) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:48:28.248+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:48:28.250+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:28.249+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:48:28.266+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:28.266+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:48:28.266+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:28.266+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:48:28.267+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:28.267+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:48:28.267+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:28.267+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:48:28.634+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:28.633+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:48:28.634+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:28.634+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:48:28.635+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:28.635+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:48:28.635+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:28.635+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:48:29.023+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:48:29.030+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:29.029+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:48:29.050+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:29.049+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:48:29.060+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.817 seconds
[2025-03-01T23:48:59.409+0000] {processor.py:186} INFO - Started process (PID=18679) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:48:59.411+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:48:59.412+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:59.412+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:48:59.434+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:59.434+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:48:59.435+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:59.435+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:48:59.436+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:59.435+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:48:59.436+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:59.436+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:48:59.775+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:59.775+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:48:59.775+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:59.775+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:48:59.776+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:59.776+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:48:59.776+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:48:59.776+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:49:00.116+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:49:00.172+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:00.172+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:49:00.180+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:00.180+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:49:00.187+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.786 seconds
[2025-03-01T23:49:30.417+0000] {processor.py:186} INFO - Started process (PID=18710) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:49:30.420+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:49:30.423+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:30.422+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:49:30.450+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:30.450+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:49:30.451+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:30.451+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:49:30.452+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:30.452+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:49:30.452+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:30.452+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:49:30.723+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:30.723+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:49:30.724+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:30.724+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:49:30.724+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:30.724+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:49:30.725+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:30.725+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:49:30.967+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:49:31.026+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:31.026+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:49:31.041+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:49:31.041+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:49:31.048+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.639 seconds
[2025-03-01T23:50:01.406+0000] {processor.py:186} INFO - Started process (PID=18741) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:50:01.408+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:50:01.409+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:01.408+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:50:01.426+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:01.426+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:50:01.427+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:01.426+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:50:01.427+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:01.427+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:50:01.427+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:01.427+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:50:01.805+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:01.805+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:50:01.806+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:01.806+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:50:01.806+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:01.806+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:50:01.807+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:01.807+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:50:02.086+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:50:02.134+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:02.134+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:50:02.143+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:02.142+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:50:02.149+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.748 seconds
[2025-03-01T23:50:32.662+0000] {processor.py:186} INFO - Started process (PID=18772) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:50:32.664+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:50:32.667+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:32.667+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:50:32.694+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:32.693+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:50:32.695+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:32.695+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:50:32.696+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:32.696+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:50:32.696+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:32.696+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:50:33.064+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:33.064+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:50:33.065+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:33.065+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:50:33.065+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:33.065+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:50:33.066+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:33.066+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:50:33.418+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:50:33.479+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:33.479+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:50:33.490+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:50:33.490+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:50:33.496+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.845 seconds
[2025-03-01T23:51:03.882+0000] {processor.py:186} INFO - Started process (PID=18803) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:51:03.883+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:51:03.884+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:03.884+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:51:03.901+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:03.901+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:51:03.902+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:03.902+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:51:03.902+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:03.902+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:51:03.902+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:03.902+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:51:04.225+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:04.225+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:51:04.226+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:04.225+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:51:04.226+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:04.226+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:51:04.227+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:04.226+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:51:04.588+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:51:04.645+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:04.644+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:51:04.657+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:04.657+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:51:04.666+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.790 seconds
[2025-03-01T23:51:35.103+0000] {processor.py:186} INFO - Started process (PID=18834) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:51:35.107+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:51:35.110+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:35.109+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:51:35.148+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:35.148+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:51:35.150+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:35.150+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:51:35.151+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:35.151+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:51:35.152+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:35.152+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:51:35.491+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:35.491+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:51:35.491+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:35.491+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:51:35.492+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:35.492+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:51:35.492+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:35.492+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:51:35.787+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:51:35.860+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:35.860+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:51:35.877+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:51:35.876+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:51:35.885+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.793 seconds
[2025-03-01T23:52:06.460+0000] {processor.py:186} INFO - Started process (PID=18865) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:52:06.463+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:52:06.464+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:06.464+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:52:06.484+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:06.484+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:52:06.484+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:06.484+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:52:06.485+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:06.485+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:52:06.485+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:06.485+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:52:06.850+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:06.850+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:52:06.851+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:06.851+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:52:06.851+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:06.851+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:52:06.852+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:06.852+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:52:07.125+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:52:07.189+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:07.189+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:52:07.199+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:07.199+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:52:07.205+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.754 seconds
[2025-03-01T23:52:37.313+0000] {processor.py:186} INFO - Started process (PID=18896) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:52:37.315+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:52:37.317+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:37.316+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:52:37.350+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:37.350+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:52:37.351+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:37.351+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:52:37.352+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:37.352+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:52:37.353+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:37.353+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:52:37.735+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:37.735+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:52:37.736+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:37.735+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:52:37.736+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:37.736+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:52:37.736+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:37.736+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:52:38.137+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:52:38.205+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:38.205+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:52:38.221+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:52:38.220+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:52:38.228+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.924 seconds
[2025-03-01T23:53:08.921+0000] {processor.py:186} INFO - Started process (PID=18927) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:53:08.923+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:53:08.925+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:08.924+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:53:08.963+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:08.962+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:53:08.964+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:08.964+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:53:08.965+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:08.965+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:53:08.965+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:08.965+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:53:09.403+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:09.403+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:53:09.404+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:09.404+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:53:09.404+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:09.404+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:53:09.405+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:09.405+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:53:09.723+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:53:09.786+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:09.786+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:53:09.801+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:09.801+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:53:09.809+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.906 seconds
[2025-03-01T23:53:40.180+0000] {processor.py:186} INFO - Started process (PID=18983) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:53:40.182+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:53:40.183+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:40.183+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:53:40.212+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:40.212+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:53:40.214+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:40.213+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:53:40.214+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:40.214+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:53:40.215+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:40.215+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:53:40.834+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:40.833+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:53:40.834+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:40.834+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:53:40.835+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:40.835+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:53:40.835+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:40.835+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:53:41.317+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:53:41.404+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:41.404+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:53:41.434+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:53:41.433+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:53:41.449+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 1.276 seconds
[2025-03-01T23:54:11.783+0000] {processor.py:186} INFO - Started process (PID=19039) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:54:11.787+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:54:11.789+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:11.788+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:54:11.846+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:11.846+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:54:11.848+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:11.848+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:54:11.851+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:11.850+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:54:11.851+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:11.851+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:54:12.280+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:12.280+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:54:12.281+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:12.281+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:54:12.281+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:12.281+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:54:12.282+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:12.282+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:54:12.616+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:54:12.681+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:12.681+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:54:12.694+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:12.694+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:54:12.704+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.933 seconds
[2025-03-01T23:54:42.948+0000] {processor.py:186} INFO - Started process (PID=19070) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:54:42.950+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:54:42.952+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:42.951+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:54:42.982+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:42.981+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:54:42.983+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:42.983+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:54:42.984+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:42.984+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:54:42.985+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:42.985+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:54:43.311+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:43.311+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:54:43.311+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:43.311+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:54:43.312+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:43.312+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:54:43.312+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:43.312+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:54:43.623+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:54:43.705+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:43.705+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:54:43.721+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:54:43.721+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:54:43.729+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.795 seconds
[2025-03-01T23:55:14.141+0000] {processor.py:186} INFO - Started process (PID=19101) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:55:14.143+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:55:14.144+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:14.143+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:55:14.185+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:14.184+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:55:14.186+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:14.186+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:55:14.187+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:14.187+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:55:14.187+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:14.187+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:55:14.548+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:14.548+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:55:14.549+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:14.549+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:55:14.549+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:14.549+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:55:14.549+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:14.549+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:55:14.847+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:55:14.928+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:14.928+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:55:14.947+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:14.947+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:55:14.955+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.825 seconds
[2025-03-01T23:55:45.629+0000] {processor.py:186} INFO - Started process (PID=19132) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:55:45.632+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:55:45.634+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:45.633+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:55:45.674+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:45.674+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:55:45.676+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:45.676+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:55:45.677+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:45.677+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:55:45.677+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:45.677+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:55:46.021+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:46.021+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:55:46.022+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:46.022+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:55:46.023+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:46.022+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:55:46.023+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:46.023+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:55:46.277+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:55:46.345+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:46.345+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:55:46.361+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:55:46.361+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:55:46.371+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.756 seconds
[2025-03-01T23:56:16.858+0000] {processor.py:186} INFO - Started process (PID=19163) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:56:16.860+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:56:16.862+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:16.861+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:56:16.895+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:16.895+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:56:16.896+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:16.896+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:56:16.897+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:16.897+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:56:16.898+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:16.898+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:56:17.226+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:17.226+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:56:17.227+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:17.226+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:56:17.227+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:17.227+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:56:17.227+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:17.227+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:56:17.564+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:56:17.640+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:17.640+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:56:17.654+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:17.654+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:56:17.665+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.814 seconds
[2025-03-01T23:56:47.848+0000] {processor.py:186} INFO - Started process (PID=19194) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:56:47.850+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:56:47.853+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:47.852+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:56:47.878+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:47.878+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:56:47.880+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:47.880+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:56:47.881+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:47.881+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:56:47.881+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:47.881+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:56:48.180+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:48.179+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:56:48.180+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:48.180+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:56:48.181+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:48.181+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:56:48.181+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:48.181+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:56:48.504+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:56:48.581+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:48.581+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:56:48.600+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:56:48.599+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:56:48.608+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.770 seconds
[2025-03-01T23:57:18.700+0000] {processor.py:186} INFO - Started process (PID=19225) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:57:18.702+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:57:18.703+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:18.703+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:57:18.723+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:18.723+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:57:18.724+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:18.724+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:57:18.725+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:18.725+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:57:18.725+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:18.725+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:57:19.018+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:19.018+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:57:19.018+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:19.018+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:57:19.019+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:19.019+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:57:19.019+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:19.019+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:57:19.266+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:57:19.332+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:19.332+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:57:19.342+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:19.342+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:57:19.349+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.658 seconds
[2025-03-01T23:57:49.588+0000] {processor.py:186} INFO - Started process (PID=19256) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:57:49.591+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:57:49.592+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:49.592+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:57:49.615+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:49.614+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:57:49.616+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:49.616+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:57:49.617+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:49.617+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:57:49.617+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:49.617+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:57:49.964+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:49.964+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:57:49.965+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:49.964+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:57:49.965+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:49.965+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:57:49.965+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:49.965+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:57:50.231+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:57:50.312+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:50.312+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:57:50.328+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:57:50.328+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:57:50.337+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.763 seconds
[2025-03-01T23:58:08.405+0000] {processor.py:186} INFO - Started process (PID=19267) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:58:08.407+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:58:08.408+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:08.408+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:58:08.415+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:08.414+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/Lab1_ETL.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/Lab1_ETL.py", line 85
    ["Time Series (Daily)"]:
                           ^
SyntaxError: invalid syntax
[2025-03-01T23:58:08.415+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:58:08.435+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.050 seconds
[2025-03-01T23:58:13.479+0000] {processor.py:186} INFO - Started process (PID=19272) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:58:13.481+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:58:13.483+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:13.482+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:58:13.517+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:13.517+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:58:13.519+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:13.518+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:58:13.520+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:13.520+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:58:13.520+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:13.520+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:58:14.000+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:14.000+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:58:14.001+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:14.001+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:58:14.001+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:14.001+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:58:14.002+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:14.002+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:58:14.308+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:58:14.315+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:14.315+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:58:14.357+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:14.357+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:58:14.371+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.898 seconds
[2025-03-01T23:58:44.458+0000] {processor.py:186} INFO - Started process (PID=19303) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:58:44.461+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:58:44.463+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:44.462+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:58:44.488+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:44.488+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:58:44.489+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:44.489+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:58:44.490+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:44.490+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:58:44.490+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:44.490+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:58:45.010+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:45.010+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:58:45.011+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:45.011+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:58:45.012+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:45.011+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:58:45.012+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:45.012+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:58:45.418+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:58:45.541+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:45.541+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:58:45.563+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:58:45.563+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:58:45.579+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 1.132 seconds
[2025-03-01T23:59:15.832+0000] {processor.py:186} INFO - Started process (PID=19334) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:59:15.835+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:59:15.836+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:15.835+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:59:15.857+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:15.857+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:59:15.858+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:15.858+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:59:15.859+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:15.859+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:59:15.859+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:15.859+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:59:16.323+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:16.323+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:59:16.324+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:16.324+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:59:16.324+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:16.324+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:59:16.324+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:16.324+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:59:16.639+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:59:16.700+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:16.700+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:59:16.712+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:16.712+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:59:16.719+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 0.901 seconds
[2025-03-01T23:59:46.815+0000] {processor.py:186} INFO - Started process (PID=19365) to work on /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:59:46.819+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/Lab1_ETL.py for tasks to queue
[2025-03-01T23:59:46.820+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:46.820+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:59:46.848+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:46.848+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:59:46.850+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:46.849+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:59:46.851+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:46.851+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:59:46.851+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:46.851+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:59:47.407+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:47.407+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-01T23:59:47.408+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:47.408+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.12.5-linuxkit-aarch64-with-glibc2.36
[2025-03-01T23:59:47.409+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:47.408+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-01T23:59:47.409+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:47.409+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-01T23:59:47.737+0000] {processor.py:925} INFO - DAG(s) 'Lab1_ETL' retrieved from /opt/airflow/dags/Lab1_ETL.py
[2025-03-01T23:59:47.797+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:47.797+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2025-03-01T23:59:47.808+0000] {logging_mixin.py:190} INFO - [2025-03-01T23:59:47.808+0000] {dag.py:4156} INFO - Setting next_dagrun for Lab1_ETL to 2025-03-01 02:30:00+00:00, run_after=2025-03-02 02:30:00+00:00
[2025-03-01T23:59:47.817+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/Lab1_ETL.py took 1.017 seconds
